app.py

```
@@ -0,0 +1,16 @@
+from flask import Flask, jsonify, request
+from game import *
+app = Flask(__name__)
+
+
+
+@app.route('/expressions', methods=['GET'])
+def get_expressions():
+    num_cnt = int(request.args.get('num_cnt', 4))
+    level = int(request.args.get('level', 2))
+
+    expr = generate(num_cnt, level)
```

Ruff (0.11.9)

`generate` may be undefined, or defined from star imports

(F405)